### DISCUSSION TITLE
Adding embeds/content for tags

### DISCUSSION THREAD

**Discussion by haotian2006** [haotian2006/MyDocs#27] [Ideas]
When you send a tag link to Discord it will display the tag contents in the embed but rn I'm unsure how to apply the md/mdx to the embed. I'm thinking about generating an image for each tag automatically and displaying it. Any ideas @cyrus01337?

**Comment by haotian2006** [comment]
![image](https://github.com/user-attachments/assets/7844c462-8b48-4b89-b972-fb4d5f66d4ea)
well it works but it takes like 15 seconds to generate the images
![image](https://github.com/user-attachments/assets/1898f326-e256-48cd-9474-0cef23f437b3)

**Reply by haotian2006** [reply]
and it doesn't support mdx

**Comment by cyrus01337** [comment]
This is very new to me, I'll look into this separately though image generation is quite the cool approach,

**Reply by haotian2006** [reply]
Also I decided to just make a workflow file that will generate all the images and store it in a public folder on push to main

**Reply by cyrus01337** [reply]
Excellent, gotta love automation! Neat trick!
